README.md

```diff
@@ -52,32 +70,12 @@ Steps
 - seeding mask (Scilpy)
 - particle filter tracking (Scilpy)
 
-Singularity
------------
-The singularity is available on
-[singularity-TractoFlow](https://github.com/scilus/singularity-tractoflow) repository
- 
-Requirements
-------------
-
-- [Nextflow](https://www.nextflow.io)
-
-The file _singularity_tractoflow.def_ contains most of the requirements commands.
-
-- Scilpy
-- Dipy
-- Mrtrix3
-- ANTS
-- FSL with the eddy_openmp
-[patch](https://fsl.fmrib.ox.ac.uk/fsldownloads/patches/eddy-patch-fsl-5.0.9/centos6/)
-
-Notes
------
-
-The _scilpy/scripts_ folder should be in your PATH environment variable if
-Singularity is not used.
-
 Usage
 -----
 
 See *USAGE* or run `nextflow run main.nf --help`
+
+References
+----------
+
+To refer to TratoFlow, please see `References` section on [TractoFlow documentation](https://tractoflow-documentation.readthedocs.io)
```

Comment:
You could also link directly to the References page (https://tractoflow-documentation.readthedocs.io/en/latest/reference/references.html)